idaes_ui/fv/fsvis.py

```diff
@@ -16,15 +16,27 @@
 # stdlib
 from collections import namedtuple
 from pathlib import Path
+import os
 import sys
 import time
 from typing import Optional, Union, Dict, Tuple
 import webbrowser
+import io
 
 # package
 from idaes import logger
 from .model_server import FlowsheetServer
 from . import persist, errors
+from IPython.display import Image as IPythonImage, display
+from IPython.display import SVG
+
+from selenium import webdriver
```

Comment:
How hard would it be to make this an optional dependency, i.e. only load it if they want a screenshot?